docs/contributing/documentation.md

```diff
@@ -190,16 +194,38 @@ There are some special cases that appear within the Enact framework. One example
 * `@param` should be repeated for each parameter. The type(s) is set within curly braces, followed by the name of the parameter. A short description of the parameter follows. Default parameter values are documented by including them within square brackets in the name position as shown above.
 * `@returns` indicates the type of the return. Functions that have no return should be documented as returning `{undefined}` to keep ESLint happy.
 
+## The Enact Voice
+
+Enact docs should strive to have a common voice.  They should be concise, informative and, as appropriate, a bit playful.  Don't be afraid to inject a little bit of fun into what could otherwise be dry reading.
+
+Tips:
+
+* Use the active voice when writing docs.
+* Descriptions should focus on what the componet or property provides.
+* Only document the `true` state for boolean propertues unless the `false` state's operation is unclear.
+
+Good:
+
+* A Moonstone-styled button with built-in support for ...
+* The color of the underline beneath button's content.
```

Comment:
In more recent components, I've attempted to use a short (< 80 characters) description for a prop on the first line and then expand on it in subsequent lines if necessary. I think this forces us to succinctly describe the feature for the user to enable them to quickly decide if they need it without wading through the full body.

As a result, the short description (like this case) isn't a sentence and so a period at the end isn't appropriate.

* Should we follow the approach I've outline above (one line summary + longer body as needed)?
* Should we
  * always use an ending period,
  * omit ending punctuation altogether, or
  * restrict it to when using a complete sentence

Reply:
This is a considerate idea. I like it. Should the next line of the description (the longer part) be separated by a blank line to begin a new paragraph? If not, it will run-in as the next sentence of the docs, rather than a new paragraph.

Reply:
Yes. I've always included a blank line between the short desc and body.

```
/**
 * Short description
 *
 * longer body
 *
 * @pragmas
 */
```

Reply:
The only problem with not adding the period is that some descriptions have it and some done.  I Think keeping it to a sentence is most useful.  Even a terse sentence deserves a period.

Opens the ExpandableItem to reveal its contents

That's still a sentence.  The subject is implied, but all the parts are there.

Reply:
But the example isn't a sentence

> The color of the underline beneath button's content.

Should it be reworded to be one?

> Sets the color of the underline beneath button's content.

Seems like we'd have a lot of "Sets ..." just so we can add a period.

Reply:
I don't think we need to get too hung up on whether something is a proper sentence or not.  Just stick a period on it and make me happy. :P  Fragments are fine for the short description.  We definitely don't want 'sets', I think.  Because props don't really -do- things, they -are- things (in most contexts).